cumulus/parachains/integration-tests/emulated/tests/bridges/bridge-hub-rococo/src/tests/snowbridge.rs

```diff
@@ -253,6 +254,17 @@ fn send_token_from_ethereum_to_penpal() {
 		(PenpalASender::get(), INITIAL_FUND),
 	]);
 
+	PenpalA::execute_with(|| {
+		<PenpalA as Chain>::System::set_storage(
```

Comment:
@claravanstaden remove `unwrap` and use `assert_ok!(` instead please

Reply:
Done in https://github.com/paritytech/polkadot-sdk/pull/3570/commits/4c2ca7b0d4543e44486ec865fe3306b41dce1acf.